src/components/breadcrumb/daikin-breadcrumb.ts

```diff
@@ -9,6 +9,24 @@ import { createRef, ref, type Ref } from "lit/directives/ref.js";
 import tailwindStyles from "../../tailwind.css?inline";
 import type { DaikinBreadcrumbItem } from "../breadcrumb-item/daikin-breadcrumb-item";
 
+/**
+ * You can use breadcrumb component and breadcrumb-item component to display the current location within a hierarchy. And allow user going back to states higher up in the hierarchy.
```

Comment:
When referring to components in a document, please use the format `daikin-*` to linkify it.

```suggestion
 * `daikin-breadcrumb` is a component for representing a breadcrumb list, and is used together with the `daikin-breadcrumb-item` component.
 *
 * Hierarchy:
 * - `daikin-breadcrumb` > `daikin-breadcrumb-item`
```